website/docs/cert-manager.md

```diff
@@ -0,0 +1,134 @@
+---
+id: cert-manager
+title: Injecting Certificates from Cert Manager
+---
+
+Gatekeeper controller generates by default and rotates a self signed CA and TLS certificate. The CA is injected into the admission webhooks, the TLS secret is mounted into the controller pod. This tutorial is about using [cert-manager](https://cert-manager.io/docs/) to manage the CA and the certificate.
+
+## Generating issuers and certificates
+
+We will create a simple self signed CA certificate with cert-manager. From this CA, we will create a certificate that we will associate to the controller.
```

Comment:
I think it's fine to reference cert-manager as a thing that can generate certificates, but we should avoid coupling our docs with their UI.

To make this more agnostic, maybe use shell commands to generate the certificate?

Reply:
Aligned.